source/_integrations/androidtv_remote.markdown

```diff
@@ -570,6 +572,11 @@ cards:
 {% enddetails %}
 
 
+## Removing the integration
```

Comment:
Nice addition! I would move it to the end of the page, as it marks the end of the integration's life cycle.